server/src/Metrics.hs

```diff
@@ -4,7 +4,10 @@ import           Prometheus
 import qualified Network.HTTP.Types as Http
 import qualified Data.ByteString.Char8 as BS8
 
-type RequestCounter = Vector (String, String) Counter
+type MethodLabel = String
+type StatusLabel = String
+
+type RequestCounter = Vector (MethodLabel, StatusLabel) Counter
```

Comment:
Looks better @fatho! Again, as an outsider, I wonder if `Method` is `HttpMethod` or access method (read / write)?